src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -36,7 +36,7 @@
 			<div ref="scroller"
 				class="left-sidebar__list"
 				@scroll="debounceHandleScroll">
-				<AppNavigationCaption v-if="isSearching"
+				<AppNavigationCaption :class="{'hidden-visually': !isSearching}"
```

Comment:
```suggestion
				<AppNavigationCaption v-show="isSearching"
```

Reply:
This will not make it appear on screenreaders in case the user is not search which is the whole point of the PR?